### ISSUE TITLE
a large number of training data with convergence at the first epoch?

### ISSUE THREAD

**Issue by frothmoon** [junyanz/pytorch-CycleGAN-and-pix2pix#894]
Thank you for your previous work. Your tips really help me in puzzled.
But  I have a very special question, I really need your help.

I have a large number of dataset(nearly 100000 images in A), and I use almost the default parameter you set. The loss converge at the first epoch.
How should I adjust parameters and optimize results？

In addition, Can I set `--input_nc 3` and `--output_nc 1` at the same time?

![output](https://user-images.githubusercontent.com/24699400/71948540-4ca2fb00-320b-11ea-8b66-c239fdedb6a6.png)

**Comment by junyanz** [comment]
If your input has 3 channels, and the output has 1 channel, you can set `--input_nc 3` and `--output_nc 1`. Your loss curves look fine. I am not sure if they have converged. The default GAN losses only oscillate.  As you have 100K images, you can train the model for fewer epochs (e.g., 5-20 epochs)

**Comment by frothmoon** [comment]
Thank you for your advice.
